Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/PaginationPanel.java

```diff
@@ -152,6 +152,19 @@ public void setNewPagesCountValue(int newPagesCountValue) {
      *            selected items to set
      */
     public void setPaginationSelectionSelectedItems(List<Integer> paginationSelectionSelectedItems) {
+        List<MediaUnit> mediaUnits = dataEditor.getWorkpiece().getAllMediaUnitChildrenFilteredByTypePageAndSorted();
+        MediaUnit lastItem = mediaUnits.get(paginationSelectionSelectedItems.get(paginationSelectionSelectedItems.size() - 1));
+        if (!Objects.equals(this.paginationSelectionSelectedItems.get(this.paginationSelectionSelectedItems.size() - 1 ), lastItem)) {
```

Comment:
`lastItem` contains a `MediaUnit`
`this.paginationSelectionSelectedItems.get(this.paginationSelectionSelectedItems.size() - 1)` returns an `Integer`

That means the comparison in this line will _always_ return `true` (because `Integer`s are always unequal to `MediaUnit's) and therefore will _always_ the metadata panel update, even if it's not necessary.